### ISSUE TITLE
Recommendation - First Riff Meeting (wk 1)

### ISSUE THREAD

**Issue by adonahue** [rifflearning/zenhub-platform-project#141]
As a PM, I want to encourage users to use the Riff platform so that they engage more deeply in the course and have better outcomes. I want to encourage them to use Riff Video right from the start, because it correlates with better outcomes. 

** Story Acceptance Criteria ** 

This rec should display under the following conditions: 
- [x] It's week 1 of the course.
- [x] Display text: “Have your first Riff meeting with your [Peer Learning Group].”
- [x] If user has NOT*participated* in a Riff call, display as incomplete. 
- [x] When user HAS *participated* in one or more Riff calls, display as complete.

**Comment by jaedoucette** [comment]
It is currently (we think) hard to determine which week of the course we are in programmatically. The best approach we can think of is "within 1 week of when the PLG channel was created."

Not clear to us what it means to have a meeting with your team. Is it _all_ of the team, some of the team?

We think that, if the business logic defined a meeting with your team as either "everyone in my PLG channel" or "any person in my PLG channel", it should be possible to do this.

John has python code that computes the right queries, but not for the production databases. This could simplify the algorithmic portion of the task.

**Comment by brecriffs** [comment]
Seems we are getting to the core of a repeating technical problem. We need the mattermost-server to know which users attended which riff video meetings. (Or at least, when a riff video link is posted in a channel, which users clicked on it.) If we have this information, generating and displaying this recommendation seems straightforward.

**Comment by jaedoucette** [comment]
We decided that a "meeting with your group" means you joined a meeting in riff-server that contained at least one person who was a member of your PLG.

**Comment by jaedoucette** [comment]
@adonahue We aren't sure whether the text here is intended to be the final text. It seems like it would be strange to have check next to this wording.

(Doesn't make sense to say "looks like you haven't met with your team yet" if you have.)

**Comment by adonahue** [comment]
@jaedoucette @brecriffs - Your definition of a "Team meeting" is correct - t least one other person from the group will count. 

As for the text, it can read: "Have your first Riff meeting with your [Peer Learning Group]." Brackets indicate a link back to their PLG channel. LMK if adding a link is a significant difference of effort for implementation. Thanks!

**Comment by adonahue** [comment]
@jaedoucette and @brecriffs - is there any minimum participation time for this to count as a meeting? (e.g. 5 minutes?). If there is, I want to make sure I meet those requirements in testing. Thanks!

**Comment by jaedoucette** [comment]
@adonahue I don't think so. It's really just looking for whether two people were in a meeting at the same time.

**Comment by adonahue** [comment]
Verified, Chrome / Staging. Accepted as is, with a slight variation in the Recommendation text.

**Comment by juliariffgit** [comment]
@brecriffs and @jaedoucette, could one of you put a points # on this when you have a moment so we can keep tracking our velocity properly?

NEVERMIND! I see it in there.
